CMakeLists.txt

```diff
@@ -33,6 +51,16 @@ include(AddLLVM)
 include(AddMLIR)
 include(HandleLLVMOptions)
 
+if(GC_ENABLE_BINDINGS_PYTHON AND NOT MLIR_ENABLE_BINDINGS_PYTHON)
+  message(STATUS "Enabling Python API failed due to the 'MLIR_ENABLE_BINDINGS_PYTHON' for LLVM is not ON.")
```

Comment:
Wording: Failed to enable Python API ...

Reply:
Would you please also update the readme of the for the cmake command. The main branch has updated the readme now.

Reply:
fixed

Reply:
> Would you please also update the readme of the for the cmake command. The main branch has updated the readme now.

have updated the readme of main project, and I will update a more specific readme in the python dir after binding more dialects in the future